src/oscar/templates/oscar/customer/emails/commtype_product_alert_body.html

```diff
@@ -0,0 +1,30 @@
+{% extends "customer/emails/base.html" %}
+{% load i18n %}
+
+{% block body %}
+    {% autoescape off %}
```

Comment:
As with the other HTML templates, I don't think `autoescape off` is appropriate here.

Reply:
Agreed.